docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -0,0 +1,171 @@
+Build or run agent service deployments.
+
+This command group consists of a number of functionalities for building service deployments, run locally stored service deployments, and run service deployments defined in the on-chain protocol. See the appropriate subcommands for more information.
+
+
+## `autonomy deploy build`
+
+Build a service deployment.
+
+This command must be executed within a service folder. That is, a folder containing the service configuration file (`service.yaml`). The deployment will be created in the subfolder `./abci_build`.
+
+### Usage
+
+```bash
+autonomy deploy build [OPTIONS] [KEYS_FILE]
+```
+
+### Options
+
+`--o PATH`
+:   Path to output directory.
+
+`--n INTEGER`
+:   Number of agents.
+
+`--docker`
+:   Use docker as a backend.
+
+`--kubernetes`
+:   Use kubernetes as a backend.
+
+`--dev`
+:   Create development environment.
+
+`--force`
+:   Remove existing build and overwrite with new one.
+
+`--log-level [INFO|DEBUG|WARNING|ERROR|CRITICAL]`
+:   Logging level for runtime.
+
+`--packages-dir PATH`
+:   Path to packages directory (Use with `dev` mode).
+
+`--open-aea-dir PATH`
+:   Path to open-aea repo (Use with `dev` mode).
+
+`--open-autonomy-dir PATH`
+:   Path to open-autonomy repo (Use with `dev` mode).
+
+`--aev`
+:   Apply environment variable when loading service config.
+
+`--use-hardhat`
+:   Include a hardhat node in the deployment setup.
+
+`--use-acn`
+:   Include an ACN node in the deployment setup.
+
+`--image-version TEXT`
+:   Define runtime image version.
+
+`--remote`
+:   To use a remote registry.
+
+`--local`
+:   To use a local registry.
+
+`-p`
+:   Ask for password interactively.
+
+`--password PASSWORD`
+:   Set password for key encryption/decryption.
+
+`--help`
+:   Show the help message and exit.
+
+
+### Examples
+```bash
+autonomy deploy build keys.json --aev
```

Comment:
--aev is the default now, it is not required to specify it AFAIK. Also keys.json.